templates/deceasedreports/check.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+{% block title %}Deceased Participant {{ participant.participantId }} - {% endblock %}
+{% block body %}
+    <div class="page-header">
+        <h2><i class="fa fa-hourglass-o" aria-hidden="true"></i> Deceased Participant
+            <small><a href="{{ path('participant', {id: participant.participantId} ) }}">{{ participant.lastName }}, {{ participant.firstName }}</a></small>
+        </h2>
+    </div>
+
+    {% include 'deceasedreports/_review-details.html.twig' %}
+    <div class="row">
+        <div class="col-md-6">
+            <label>Has the Next of Kin requested a full withdrawal for the deceased participant?</label>
+            <div class="radio">
+                <label><input type="radio" name="full-withdrawn" value="yes"/> Yes</label>
+            </div>
+            <div class="radio">
+                <label><input type="radio" name="full-withdrawn" value="no"/> No</label>
+            </div>
+            <div style="display: none;" id="deceased_check_continue">
+                <a href="{{ path('read_deceased_report_new', {participantId: participant.participantId}) }}" type="submit" class="btn btn-primary">Continue</a>
+            </div>
+            <div style="display: none;" class="alert alert-danger" id="deceased_check_warning">
+                Please follow the withdrawal workflow for this participant.
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block pagejs %}
+    {{ encore_entry_script_tags('deceased-report') }}
+{% endblock %}
```

Comment:
I think we want our twig files to end with a new line like our PHP files for consistency.